libcodechecker/server/api/report_server.py

```diff
@@ -410,6 +410,18 @@ def get_report_path_hash(report, files):
     return hashlib.md5(report_path_hash.encode()).hexdigest()
 
 
+def filter_outstanding_reports(q):
```

Comment:
I am not sure I like the name of this method. We basically filter out reports that are either already categorized as false positive or intentional and reports that are fixed. We need a name like filter taken care of reports or filter actioned reports or something like that.